cli/io/spec_storage.py

```diff
@@ -0,0 +1,151 @@
+from cli import Cli
+
+
+class SpecStorageError(Exception):
+    pass
+
+
+class specStorage(Cli):
+    def __init__(self, primary_client):
+        super(specStorage, self).__init__(primary_client)
+        self.base_cmd = "python3 /root/SPECstorage2020/SM2020"
+        self.packages = ["matplotlib", "PyYAML", "pylibyaml"]
+        self.config = "sfs_rc"
+        self.install_dest = "/root/SPECstorage2020"
+        self.outputlog = "result"
+        self.benchmark_file = "storage2020.yml"
+
+    def install_spec_storage(self):
+        """
+        Install SPECstorage
+        """
+        try:
+            # Install prerequiste packages
+            for package in self.packages:
+                cmd = f"pip3 install {package}"
+                self.execute(sudo=True, cmd=cmd)
+            cmd = "yum install -y sshpass"
+            self.execute(sudo=True, cmd=cmd)
+
+            # Disable firewall
+            cmd = "systemctl stop firewalld"
+            self.execute(sudo=True, cmd=cmd)
+            cmd = "systemctl disable firewalld"
+            self.execute(sudo=True, cmd=cmd)
+            cmd = "setenforce 0"
+            self.execute(sudo=True, cmd=cmd)
+            cmd = "sed -i s/^SELINUX=.*$/SELINUX=disabled/ /etc/selinux/config"
+            self.execute(sudo=True, cmd=cmd)
+
+            # Add SpecStorage tool
+            cmd = "mount -t nfs -o sec=sys,nfsvers=4.1 reesi004.ceph.redhat.com:/ /tmp"
```

Comment:
Use library - https://github.com/red-hat-storage/cephci/blob/master/cli/utilities/filesys.py#L11

Reply:
Hi @vamahaja 
This is magna mount , Not ceph nfs server

As per the planning discussion with team we are planning to copy the spec storage tool from magna location